Dockerfile

```diff
@@ -49,16 +45,10 @@ USER sync-engine
 
 WORKDIR /opt/app
 
-ENV PATH="/opt/venv/bin:$PATH"
-
 COPY --chown=sync-engine:sync-engine ./ ./
 RUN python3 -m pip install pip==24.0 virtualenv==20.25.1 && \
   python3 -m virtualenv /opt/venv && \
-  /opt/venv/bin/python3 -m pip install --no-deps -r requirements/prod.txt -r requirements/test.txt && \
+  /opt/venv/bin/python3 -m pip install --no-cache --no-deps -r requirements/prod.txt -r requirements/test.txt && \
```

Comment:
We don't need to store wheels in the image since they will be only used once for install and it increases image size.